frontend2/src/app/(home)/CanvasContainer.tsx

```diff
@@ -8,29 +8,27 @@ import { flex } from "styled-system/patterns";
 
 import { Skeleton } from "@/components/Skeleton";
 import { PIXELS_PER_SIDE } from "@/constants/canvas";
+import { APP_CONFIG } from "@/constants/config";
+import { useAptosNetworkState } from "@/contexts/wallet";
 import { isServer } from "@/utils/isServer";
 import { getPixelArrayFromImageElement } from "@/utils/tempCanvas";
 
 import { CanvasOverlay } from "./CanvasOverlay";
 
-const CANVAS_IMAGE_URL = process.env.NEXT_PUBLIC_CANVAS_IMAGE_URL;
-
 export function CanvasContainer() {
   const [canvasContainer, containerBounds] = useMeasure();
+  const { network } = useAptosNetworkState();
   const { height, width } = containerBounds;
   const hasSize = Boolean(height && width);
 
   const [baseImage, setBaseImage] = useState<Uint8ClampedArray>();
 
   useEffect(() => {
```

Comment:
We should add `network` as a dependency of this `useEffect` so that the base image gets updated when the network changes